jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -245,10 +247,10 @@ public JibContainerBuilder addEnvironmentVariable(String name, String value) {
    * <p>This is similar to <a href="https://docs.docker.com/engine/reference/builder/#volume">{@code
    * VOLUME} in Dockerfiles</a>.
    *
-   * @param volumes the list of directory paths on the container filesystem to set as volumes
+   * @param volumes the set of directory paths on the container filesystem to set as volumes
```

Comment:
you could remove the collection type and leave it as _the directory paths on the …_